components/layout_thread/lib.rs

```diff
@@ -277,7 +277,9 @@ impl LayoutThreadFactory for LayoutThread {
                     layout.start();
                 }, reporter_name, sender, Msg::CollectReports);
             }
-            let _ = content_process_shutdown_chan.send(());
+            if let Some(content_process_shutdown_chan) = content_process_shutdown_chan {
```

Comment:
We only listen on the receiver if we spawned a new script thread (or process) for the new pipeline; `about:blank` always lives in its parent's script thread. I could create a channel and immediately drop the receiver, but that seemed pointless.